modules/trusted-firmware-m/Kconfig.tfm

```
@@ -310,6 +310,17 @@ config TFM_MCUBOOT_PATH_DOWNLOAD
 
 endchoice
 
+config TFM_ETHOS_DRIVER_PATH_LOCAL
+	string
+	prompt "Path to locally available Ethos-U driver"
+	default "${ZEPHYR_HAL_ETHOS_U_MODULE_DIR}"
+	help
+	  Path to locally available Ethos-U driver to be used for TF-M builds.
+	  By default we will use Zephyr's Ethos-U driver which is present in
```

Maybe change the tense?
```suggestion
	  By default Zephyr's Ethos-U driver will be used. It is present in
```